APMrover2/AP_MotorsUGV.cpp

```diff
@@ -591,6 +614,60 @@ void AP_MotorsUGV::output_skid_steering(bool armed, float steering, float thrott
     output_throttle(SRV_Channel::k_throttleRight, 100.0f * motor_right);
 }
 
+// output for custom configurations
+void AP_MotorsUGV::output_custom_config(bool armed, float steering, float throttle, float lateral)
+{
+    // exit immediately if the frame type is set to UNDEFINED
+    if (rover.get_frame_type() == FRAME_TYPE_UNDEFINED) {
+        return;
+    }
+
+    if (armed) {
+
+        // clear and set limits based on input
+        set_limits_from_input(armed, steering, throttle);
+
+        // constrain steering
+        steering = constrain_float(steering, -4500.0f, 4500.0f);
+
+        // reverse steering when throttle is negative
+        if (is_negative(throttle)) {
```

Comment:
Although I suspect it's an existing issue with the omni rover code, I don't think we need this reversing of steering when throttle is negative.

It can be tested on a real vehicle by setting [PILOT_STEER_TYPE](http://ardupilot.org/rover/docs/rover-steering-input-type-and-reversing-behaviour.html) = 3 and then putting it into manual mode.  If setup correctly you will find the vehicle will turn in the same direction (i.e. clockwise when right steering input is provided by the user) when throttle is forward or backwards.  I.e. it will rotate in the same direction regardless of whether it is moving forward or backwards.

Reply:
You're right, setting PILOT_STEER_TYPE to 3 was enough.